jest-preset.front.js

```diff
@@ -78,7 +78,12 @@ module.exports = {
   transformIgnorePatterns: [
     'node_modules/(?!(react-dnd|dnd-core|react-dnd-html5-backend|@strapi/design-system|@strapi/icons|fractional-indexing)/)',
   ],
-  testMatch: ['**/tests/**/?(*.)+(spec|test).[jt]s?(x)'],
+  testMatch: [
+    '**/admin/src/**/tests/**/?(*.)+(spec|test).[jt]s?(x)',
+    '**/admin/src/**/__tests__/**/?(*.)+(spec|test).[jt]s?(x)',
+    '**/helper-plugin/src/**/tests/**/?(*.)+(spec|test).[jt]s?(x)',
+    '**/helper-plugin/src/**/__tests__/**/?(*.)+(spec|test).[jt]s?(x)',
+  ],
```

Comment:
Could you elaborate why you need that? I think it would be better to set testMatch for every package?